test/rekt/resources/kafkacat/kafkacat_test.go

```diff
@@ -0,0 +1,138 @@
+/*
+Copyright 2021 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kafkacat
+
+import (
+	"os"
+
+	"knative.dev/reconciler-test/pkg/manifest"
+)
+
+func Example_min() {
```

Comment:
Nit: Why not `ExampleMin` and `ExampleFull` ? AFAIK, that's the "case" we use.

Reply:
More important: What's the purpose of these functions? It's not a test function but it's in a file with `_test.go` suffix. I don't see this function being called anywhere. Is it some temporary stuff?

Reply:
This is correct. See https://blog.golang.org/examples#TOC_4. 

I actually really like this tests since they help developing/debugging the YAML templates. `go test ./...` runs these functions.

Reply:
Oh. Thanks. Didn't know that.